### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by mirronake** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#142] [Q&A] [answered]
Hi! I have a couple of questions:
1. How to add any map to favorites using the API
2. Is it possible to put 'Like' on the map via the API (it can be put in the rink itself on the Creative Mode map and nowhere else)
3. Is it possible to collect offline purchases made via Xbox PSN using the API? In order not to log into the game via the console to receive a purchase made from Xbox and other platforms.

**Comment by entraptaa** [comment, marked as answer]
> How to add any map to favorites using the API
[Discovery Service: Add Favorite Link](https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/EpicGames/FN-Discovery-Service/Links/Favorites/Add.md)

> Is it possible to collect offline purchases made via Xbox PSN using the API? In order not to log into the game via the console to receive a purchase made from Xbox and other platforms.

[RedeemRealMoneyPurchases](https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/Fortnite/MCP/operations/RedeemRealMoneyPurchases.md)

Not sure about your second question.

**Reply by mirronake** [reply]
And one more question
Is it possible to see which map/mode a player is playing on in Fortnite? For example, I enter a map in Fortnite and through the API, I can see which map I am playing on or at least its name.

**Reply by LeleDerGrasshalmi** [reply]
you are not allowed to use matchmaking apis
